test/systemtests/trigger_test.go

```diff
@@ -248,7 +248,7 @@ func (s *systemtestSuite) TestTriggerNetpluginDisconnect(c *C) {
 
 func (s *systemtestSuite) TestTriggerNodeReload(c *C) {
 	// can not run this test on docker 1.10 & k8s
-	if os.Getenv("CONTIV_DOCKER_VERSION") == "1.10.3" || s.basicInfo.Scheduler == "k8" {
+	if os.Getenv("CONTIV_DOCKER_VERSION") == "1.10.3" || s.basicInfo.Scheduler == kubeScheduler {
```

Comment:
I can remove the check for docker (will do it as a separate PR soon - want to keep this PR solely for kubeadm test cluster bringup).